pkg/data/data_test.go

```diff
@@ -0,0 +1,29 @@
+package data
+
+import "testing"
+
+func TestData(t *testing.T) {
+	client := &Client{
+		ApiKey:    "",
```

Comment:
Kindly borrow inspiration from how tests are defined in the other packages. One should be able to run these tests without changing the code:
https://github.com/EdwinWalela/africastalking-go/blob/0b1a0b87f1315f2b28f5764b745f424486509ded/pkg/sms/sms_test.go#L11

Reply:
okay